### DISCUSSION TITLE
Version 1.7 - Keyboard Button and Card Scaling

### DISCUSSION THREAD

**Discussion by PRProd** [PRProd/HA-Firemote#9] [Announcements]
### Yet another release!

 - Added a keyboard button to simplify text input
 - Added the ability to change the card size (Feature request #7 @jscherry)
 - Minor UI adjustments

Enjoy!!

<hr /><em>This discussion was created from the release <a href='https://github.com/PRProd/HA-Firemote/releases/tag/v1.7.0'>Keyboard Button and Card Scaling</a>.</em>

**Comment by toddsay** [comment]
Nice job (again), and so many quick improvements. That keyboard would be very handy for app logins and entering long passwords. 👍

**Comment by PRProd** [comment]
### [Version 1.7.1](../releases/tag/v1.7.1)
Home Assistant Version 2022.11 made adjustments to the default styling of cards, adding a border around each.  This border detracted from the intended "realistic" look of the Firemote card, so [Version 1.7.1](../releases/tag/v1.7.1) contains a minor style change to override the new default Home Assistant style.

### Other News
You may have noticed that before I was extremely active, but now I seem to have gone silent for a couple of weeks.  It's true.  I needed some time to relax and focus on other things for a while.  I felt that this was the perfect time to hold off on big changes, while waiting for approval from the HACS folks about including the Firemote card in their default list.  
[Click here to view the HACS Pull Request](https://github.com/hacs/default/pull/1540)

### How can I help?
Are there any more (non-sideloaded) app shortcut buttons that you'd like to see added?  Other than #6 , are there any other feature requests you'd like to have considered?  Do you have any praise or hate for the project that you feel like adding to the conversation?  Please share those things here!  The floor is yours.

**Comment by PRProd** [comment]
### News Update
Firemote is now a part of the HACS default repository list!  I'm very excited about that!

**Comment by PRProd** [comment]
### 42 Products?  Oh my!
29 Smart TV models, 10 stick models, and 3 cube models!  This was a surprising revelation for me today when I discovered the [Amazon Fire TV Comparison table](https://developer.amazon.com/docs/fire-tv/device-specifications-comparison-table.html).  As of today, 6 discreet models are supported by Firemote.  I guess this means that I have the privilege of continuing to work on this project for a long long time! :-) 

Yikes... maybe I need to start a collection fund so I can get my hands on more of these devices for testing!  Maybe hold a sign on a street corner?  People hand Amazon devices out to beggars, right?  LOL!

I'm starting to think that even with the [README.md](../blob/main/README.md) file, and these discussions, it might also be useful for me to start a wiki for the project here in Github.  Big projects require big documentation, right?
